fragments/k8s/core/README.md

```diff
@@ -439,22 +429,19 @@ to determine fault tolerance.
    kubernetes-master is missing its resources, download them from the resources
    section of the sidebar [here](https://jujucharms.com/u/containers/kubernetes-master/)
    and install them by running
-   `juju attach kubernetes-master kubernetes=/path/to/kubernetes-master.tar.gz`.
+   `juju attach kubernetes-master kubernetes=/path/to/resource`.
```

Comment:
Again, there no single resource called kubernetes so this should be <resource-name>=/path/to/resource